### DISCUSSION TITLE
Regenerate certificate

### DISCUSSION THREAD

**Discussion by tpalus** [TabularEditor/TabularEditor3#1304] [Learn]
Hi, Is there way to regenerate certificate? I set only first name in easy generator :/

**Comment by otykier** [comment]
Hi @tpalus 
You should be able to change your name under [My Courses > Profile](https://learn.easygenerator.com/#/profile). The certificate should reflect the name change the following day.

**Reply by tpalus** [reply]
Nice! It works :)
